.gitignore

```diff
@@ -19,3 +19,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+*lock*
```

Comment:
`*.log` ?

I personally don't mind the "lock" files going to GitHub as they're be used in professional development teams.